File: 프로그래머스/lv0/120853. 컨트롤 제트/컨트롤 제트.swift
import Foundation

func solution(_ s:String) -> Int {
    let array = s.components(separatedBy: " ")
    var lastInputNum = 0
    var answer = 0
    for i in array {
        if let num = Int(i) {
            lastInputNum = num
            answer += num
            continue
        }
        answer -= lastInputNum
    }
    return answer
}

File: 프로그래머스/lv0/120853. 컨트롤 제트/컨트롤 제트.js
function solution(s) {
    const arr = s.split(' ');
    let answer = 0;
    let temp = 0;
    for (e of arr) {
        if (e === 'Z') {
            answer -= temp;
            continue;
        }
        temp = Number(e);
        answer += Number(e);
    }
    return answer;
}